cytopipe/cli/args.py

```diff
@@ -0,0 +1,98 @@
+# ------------------------------------------------------------
+# args.py
+#
+# Responsible for centralizing and handling all user inputted
+# parameters for cytopipe.
+#
+# Contains different modes with their respected parameters
+# ------------------------------------------------------------
+import argparse
+
+
+def parse_init_args(args_list: list) -> argparse.Namespace:
+    """Parses user inputs for setting up files
+
+    Parameters
+    ----------
+    params : list
+        User defined input parameters
+
+    Returns
+    -------
+    argparse.Namespace
+        parsed cli parameters
+
+    Attributes:
+    -----------
+    mode : str
+        name of the mode used
+    data : list[str]
+        list of plate data paths
+    metadata : str
+        Path to meta data directory
+    barcode : str
+        Path to barcode file
+    platemaps : str
+        Path to platemap file
+
+    """
+
+    # CLI inputs
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "mode",
+        choices=["init"],
+        help="Init mode sets up the all files for cytopipe processing",
+    )
+    parser.add_argument(
+        "-d", "--data", nargs="+", required=True, help="list of plate data files"
```

Comment:
>this has the potential to get ugly (e.g. what happens if someone has 1,000 plates to process?). 
I think it is ok as-is now, for prototyping. But we should consider being able to pass a file into the --data flag.

In this senario, 1,000 plates means 1,000 plates along with it's metadata  and bar code files, correct? That should not be a problem as the `--data` parameter takes in a list of plates. This is denoted by the `nargs="+"`. 

For example, users can use wildcards to capture all 1,000s plates:
```
cytopipe init -d *_plate.sqlite -m metdata_dir/ -b barcodes.txt
```

Also, 
>ohh, I see. metadata and barcode are separate flags. ￼ maybe data should be an optional parameter then? To be used only if you want to process a portion of the data?

I ended up removing the `-p` option because I realize that platemaps are stored within the metadata directory. 

This is the commit: https://github.com/WayScience/CytoPipe/pull/15/commits/d4ef1fdf05febe35c7f8a5c9703a210bb5066d50


Hopefully this answers your question.